src/core_plugins/metrics/server/lib/get_vis_data.js

```diff
@@ -0,0 +1,60 @@
+import _ from 'lodash';
+import moment from 'moment';
+import Color from 'color';
+import calculateLabel from '../../public/components/vis_editor/lib/calculate_label';
```

Comment:
sorry, i am not talking about calculateLabel, just wanted to put comment on the beginning of this file.